apps/daimo-bot/api/PaymentActionProcessor.ts

```diff
@@ -0,0 +1,235 @@
+import {
+  EAccount,
+  assert,
+  dollarsToAmount,
+  encodeRequestId,
+  formatDaimoLink,
+  generateRequestId,
+} from "@daimo/common";
+import { NeynarAPIClient } from "@neynar/nodejs-sdk";
+import dotenv from "dotenv";
+import {
+  CONNECT_FC_MESSAGE,
+  PAYMENT_CONNECT_FC_MESSAGE,
+  REQUEST_PAYMENT_MESSAGE,
+} from "./botResponses";
+import { trpcClient } from "./trpcClient";
+import { SendCastOptions, TRPCClient, WebhookEvent } from "./types";
+
+dotenv.config();
+
+assert(!!process.env.NEYNAR_API_KEY, "NEYNAR_API_KEY is not defined");
+const _neynarClient = new NeynarAPIClient(process.env.NEYNAR_API_KEY);
```

Comment:
Convention was intended so that it discourages unintended use within the class, in favor of `this.neynarClient`, to preserve use of dependency injection for things like testing.

I changed to NEYNAR_CLIENT as its a var set outside class context

Reply:
> I changed to NEYNAR_CLIENT as its a var set outside class context

Still looks like an `_`, did you miss pushing a change?

Reply:
yep, pushed